src/hooks/usePDFPreview.ts

```diff
@@ -15,7 +15,7 @@ function generateHighlights(document: TDocumentPage, documentPassageMatches: TPa
         source: document.import_id,
         selector: {
           node: {
-            index: passage.text_block_page,
+            index: passage.text_block_page - 1,
```

Comment:
How does it know this is Vespa and not opensearch?

Reply:
see slack chat https://climate-policy-radar.slack.com/archives/C02ET708QDP/p1699280107665369

Reply:
I am purposely not adding any search provider business logic into the PDF previewer as I think we need to fix this upstream from the FE. This means Vespa results will be out by -1.